pom.xml

```diff
@@ -53,15 +53,16 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <awaitility.version>3.1.6</awaitility.version>
-    <cdap.version>6.1.1</cdap.version>
+    <cdap.version>6.4.0</cdap.version>
```

Comment:
Can you clarify why we need to bump CDAP and hydrator version? If we're bumping the CDAP dependency version, we'll want to bump the minor version as well to ensure there was a working version of the plugin at the time of deployment.